### PR TITLE
#939 and #1111 VeText 502

### PR DESCRIPTION
**Pull request by jakehova**
# Description

#939 #1111 
This PR should resolve both bug tickets we were having in Twilio by returning a simple status code. Removed the twiml response and simply gave it an empty `Response` tag.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Created inbound SMS from twilio to dev inbound forwarder endpoint
- [x] Verified that no 502 error was received in Twilio's logs

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by k-macmillan — APPROVED** [review]

File: tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py [hunk 1]
```diff
@@ -37,206 +120,290 @@ def json(self):
 
 
 def mocked_requests_post_success(*args, **kwargs):
-    return MockResponse({}, 200, '<?xml version="1.0" encoding="UTF-8"?><Response></Response>')
+    return MockResponse(
+        {}, 200, '<?xml version="1.0" encoding="UTF-8"?><Response></Response>'
```
Comment (k-macmillan) [inline tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py:124]: I believe this needs to be changed now.

**Review by k-macmillan — CHANGES_REQUESTED** [review]

File: tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py (hunk 1 above)
Comment (k-macmillan) [reply tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py:124]: This appears to be for the post to vetext, not our response to Twilio.

**Review by k-macmillan — DISMISSED** [review]

**Comment by k-macmillan** [comment]
Master merge involved github actions and delivery status work. None of that applies to this lambda.

**Review by k-macmillan — APPROVED** [review]
I did not change any of this code. No conflict of interest.

**Review by ldraney — APPROVED** [review]

**Pull request merged**